ods_ci/build/install_idp.sh

```diff
@@ -446,10 +430,35 @@ function validate_user_config_file(){
   validate_user_config_fields_and_values ldap
 }
 
-if [ "${USE_OCM_IDP}" -eq 1 ]
-      then
-          perform_ocm_login
-fi
+function update_test_config(){
+    echo "Update test config file..."
+
+    AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+    AWS_ACCESS_KEY_ID=$(cat $AWS_SHARED_CREDENTIALS_FILE | grep aws_access_key_id | tr -d ' ' | cut -d '=' -f 2)
+    AWS_SECRET_ACCESS_KEY=$(cat $AWS_SHARED_CREDENTIALS_FILE | grep aws_secret_access_key | tr -d ' ' | cut -d '=' -f 2)
+    export AWS_ACCESS_KEY_ID
+    export AWS_SECRET_ACCESS_KEY
+
+    yq -i '.OCP_API_URL=env(OC_HOST)' test-variables.yml
+    yq -i '.OCP_CONSOLE_URL=env(OCP_CONSOLE)' test-variables.yml
+    yq -i '.ODH_DASHBOARD_URL=env(RHODS_DASHBOARD)' test-variables.yml
+    yq -i '.BROWSER.NAME="firefox"' test-variables.yml
+    yq -i '.S3.AWS_ACCESS_KEY_ID=env(AWS_ACCESS_KEY_ID)' test-variables.yml
+    yq -i '.S3.AWS_SECRET_ACCESS_KEY=env(AWS_SECRET_ACCESS_KEY)' test-variables.yml
+}
+
 validate_user_config_file
 check_installation
 install_identity_provider
+update_test_config
+
+echo "Performing oc login using TEST username and password"
```

Comment:
the login is performed earlier in this script with htp user. Why do we need to log in with a different one?